types/validator_set.go

```diff
@@ -360,6 +360,14 @@ func (vals *ValidatorSet) findProposer() *Validator {
 	return proposer
 }
 
+func (vals *ValidatorSet) ValidatorBlocksTheChain(address []byte) bool {
+	_, val := vals.GetByAddress(address)
+	if val == nil {
+		return false
+	}
+	return val.VotingPower > vals.TotalVotingPower()/3
```

Comment:
```suggestion
	return val.VotingPower >= vals.TotalVotingPower()/3
```

?

Reply:
We probably need a test unit for this :)